buildSrc/build.gradle

```diff
@@ -24,25 +24,26 @@ repositories {
 }
 
 dependencies {
-	implementation("commons-io:commons-io:2.11.0")
-	implementation("de.undercouch:gradle-download-task:4.1.1")
-	implementation("net.fabricmc:stitch:${project.stitch_version}")
-	implementation("org.quiltmc:quilt-enigma-plugin:${project.enigma_plugin_version}")
-	implementation('org.quiltmc:tiny-remapper:0.7.2')
-	implementation('org.quiltmc:launchermeta-parser:1.0.0')
-	implementation("com.google.guava:guava:30.1.1-jre")
-	implementation("cuchaz:enigma-cli:${project.enigma_version}")
-	implementation("cuchaz:enigma:${project.enigma_version}")
-	implementation("org.quiltmc.unpick:unpick:${project.unpick_version}")
-	implementation("org.quiltmc.unpick:unpick-format-utils:${project.unpick_version}")
-	implementation("org.quiltmc.unpick:unpick-cli:${project.unpick_version}")
-	implementation("net.fabricmc:mapping-io:0.3.0")
-	implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.13.0")
-	implementation("org.quiltmc:javadoc-draftsman:1.2.1")
+	implementation(libs.commons.io)
+	implementation(libs.download.task)
+	implementation(libs.guava)
+	implementation(libs.jackson.xml)
+	implementation(libs.launchermeta.parser)
+
+	implementation(libs.enigma)
+	implementation(libs.enigma.cli)
+	implementation(libs.enigma.plugin)
+	implementation(libs.tiny.remapper)
+	implementation(libs.stitch)
+	implementation(libs.unpick)
+	implementation(libs.unpick.cli)
+	implementation(libs.unpick.format.utils)
+	implementation(libs.mapping.io)
+	implementation(libs.javadoc.draftsman)
```

Comment:
A lot of these can be deduped with the use of bundles

Reply:
Should be added now, It's kinda lame but it's the best i can do on my own.

Reply:
There could be more but it's good enough for me haha